package.json

```diff
@@ -18,23 +19,25 @@
   },
   "homepage": "https://macosnotif.js.org",
   "devDependencies": {
-    "@babel/core": "^7.1.2",
-    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
-    "@babel/plugin-transform-template-literals": "^7.0.0",
-    "babel-loader": "^8.0.4",
+    "@babel/core": "^7.4.5",
+    "@babel/plugin-proposal-object-rest-spread": "^7.4.4",
+    "@babel/plugin-transform-template-literals": "^7.4.4",
+    "@babel/polyfill": "^7.4.4",
+    "@babel/preset-env": "^7.4.5",
+    "babel-loader": "^8.0.6",
     "babel-plugin-async-to-promises": "^1.0.5",
     "babel-plugin-transform-es3-member-expression-literals": "^6.22.0",
     "babel-plugin-transform-es3-property-literals": "^6.22.0",
-    "babel-preset-env": "^7.0.0-beta.3",
-    "core-js": "^2.5.7",
-    "css-loader": "^1.0.0",
-    "file-loader": "^2.0.0",
+    "core-js": "2.x",
```

Comment:
It might be better to leave this as `^2.5.7`, unless this is something that babel or web pack is now relying on to determine core-js version?

Reply:
This could lead to issues with 3.x being used which isn't support, so defining it as `2.x` is better.